### PR TITLE
Fix placement of ColorPicker in the editor

### LINKED ISSUE #91722: Opening a Color picker in the inspector places it outside the display border
### Tested versions

4.3 dev, 2042420bd7f43bb834316d7ca4631ab2fcfcc820

### System information

Windows 11

### Issue description

Opening a color picker dialog for the full-screened inspector (in the standard editor layout) place it outside the window, which makes it unusable:

изображение

Probably a recent regression (does not present in 4.3 dev 6)

### Steps to reproduce

^^

### Minimal reproduction project (MRP)

No need

### PR DESCRIPTION
**Pull request by Chaosus**
My attempt to Fix https://github.com/godotengine/godot/issues/91722

### CODE REVIEW

**Comment by KoBeWi** [comment]
I'm looking into that right now and BoxContainer was already using `is_visible_in_tree()`, so I don't get why the new method wouldn't work.

**Comment by Chaosus** [comment]
@KoBeWi Ok, if you find the right solution, feel free to close this PR.

File: scene/gui/box_container.cpp [hunk 1]
```diff
@@ -243,8 +243,9 @@ Size2 BoxContainer::get_minimum_size() const {
 	bool first = true;
 
 	for (int i = 0; i < get_child_count(); i++) {
-		Control *c = as_sortable_control(get_child(i));
-		if (!c) {
+		// as_sortable_control should not been used here, because is_visible_in_tree used inside that method (instead of is_visible) provides negative impact on placement for some containers (see https://github.com/godotengine/godot/issues/91722).
```
Comment (KoBeWi) [inline scene/gui/box_container.cpp:246]: I think the comment is unnecessary, it's the same case as #91708.

**Review by KoBeWi — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**